src/components/Header.js

```diff
@@ -1,7 +1,26 @@
+import React, { useState, useEffect } from 'react';
 import GitHubButton from 'react-github-btn'
 import './Header.css'
+import Toggle from './Toggle';
+import { keepTheme } from '../utils/theme';
 
 function Header() {
+
+    const [togClass, setTogClass] = useState('dark');
+    let theme = localStorage.getItem('theme');
+
+    useEffect(() =>
+    {
+        keepTheme();
+        if(localStorage.getItem('theme') === 'theme-dark')
+        {
+            setTogClass('dark')
+        }
+        else if(localStorage.getItem('theme') === 'theme-light')
+        {
+            setTogClass('light')
+        }
+    }, [theme]);
```

Comment:
```suggestion
    useEffect(() => {
        keepTheme()
        const storagedTheme = localStorage.getItem('theme')
        setTogClass(storagedTheme)
    }, [theme]);
```
And then changed the storage value to dark and light so you don't need to if/else.